modules/store/src/store.ts

```diff
@@ -93,6 +94,17 @@ export class Store<T = object>
     return (select as any).call(null, pathOrMapFn, ...paths)(this);
   }
 
+  /**
+   * Returns a signal of the provided selector.
+   * This method must be run within in injection context.
+   *
+   * @param selectorFn selector function
+   */
+  selectSignal<K>(selectorFn: (state: T) => K): Signal<K>;
+  selectSignal<K = unknown>(selectorFn: (state: T) => K): Signal<unknown> {
```

Comment:
it seems that an overload signature is not needed here:

```suggestion
  selectSignal<K>(selectorFn: (state: T) => K): Signal<K> {
```